app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="bottom-box" :class="wrapClass">
+    <div class="left-wrap">
+      <button class="button clear button--emoji" @click="toggleEmojiPicker">
+        😊
+      </button>
+      <button class="button clear button--emoji button--upload">
+        <file-upload
+          v-if="showFileUpload"
+          :size="4096 * 4096"
+          accept="image/*, application/pdf, audio/mpeg, video/mp4, audio/ogg, text/csv"
+          @input-file="onFileUpload"
+        >
+          📎
+        </file-upload>
+      </button>
+    </div>
+    <div class="right-wrap">
+      <button
+        class="button nice primary button--send"
+        :class="buttonClass"
+        @click="onSend"
+      >
+        {{ sendButtonText }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import FileUpload from 'vue-upload-component';
+
+import { REPLY_EDITOR_MODES } from './constants';
+export default {
+  name: 'ReplyTopPanel',
+  components: { FileUpload },
+  props: {
+    mode: {
+      type: String,
+      default: REPLY_EDITOR_MODES.REPLY,
+    },
+    onSend: {
+      type: Function,
+      default: () => {},
+    },
+    sendButtonText: {
+      type: String,
+      default: '',
+    },
+    showFileUpload: {
+      type: Boolean,
+      default: false,
+    },
+    onFileUpload: {
+      type: Function,
+      default: () => {},
+    },
+    showEmojiPicker: {
+      type: Boolean,
+      default: false,
+    },
+    toggleEmojiPicker: {
+      type: Function,
+      default: () => {},
+    },
+    isSendDisabled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    wrapClass() {
+      return {
+        'is-note-mode': this.mode === REPLY_EDITOR_MODES.NOTE,
+      };
+    },
+    buttonClass() {
+      return {
+        'button--note': this.mode === REPLY_EDITOR_MODES.NOTE,
+        'button--disabled': this.isSendDisabled,
+      };
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '~widget/assets/scss/variables.scss';
+@import '~widget/assets/scss/mixins.scss';
+
+.bottom-box {
+  display: flex;
+  justify-content: space-between;
+  padding: var(--space-slab) var(--space-normal);
+
+  &.is-note-mode {
+    background: var(--y-50);
+  }
+}
+
+.button {
```

Comment:
These styles are close to tailwind button styles, Didn't move it to a generic file since polluting it might cause breakages. Also we might be going with tailwind soon.